client/pages/index.tsx

```diff
@@ -7,6 +7,7 @@ import { getAllPosts } from '../lib/api'
 import Head from 'next/head'
```

Comment:
post here a screenshot how, it's in desktop and mobile :)

Reply:
i didn't created resolution for mobile screen, just md screen

![Screenshot from 2023-03-21 18-16-49](https://user-images.githubusercontent.com/106893704/226742925-24e1136b-4192-4947-bc30-1a5e0a4ff5e0.png)

![Screenshot from 2023-03-21 18-16-44](https://user-images.githubusercontent.com/106893704/226742997-ce2417d7-a215-48db-8f1c-edd72d4b15b4.png)

Reply:
Remove this messa/block on top of navbar

Reply:
the menu and stuff i still have to fix,is breaked so header as footer.

Reply:
Then it isn't done yet, is it?

Reply:
no isn't

Reply:
header and footer are also responsive now

Reply:
i changed them both, for now on the header it has a drop down that don't pop up nothing, because i was thinking to add this functionality on another pr to don't overload this pr

Reply:
Ok could you post a new screenshot :)

Reply:
ok :p

Reply:
these are header/footer on desktop size(1440):

![Screenshot from 2023-04-17 17-54-21](https://user-images.githubusercontent.com/106893704/232608559-979209b5-5593-4a70-a49a-a1e95657eeac.png)
![Screenshot from 2023-04-17 17-54-10](https://user-images.githubusercontent.com/106893704/232608603-af6811a9-8e14-4f60-bf8a-2569a2617865.png)

these are header/footer on mobel size(320):
![Screenshot from 2023-04-17 17-53-26](https://user-images.githubusercontent.com/106893704/232608613-f05a9e3c-c9df-40e6-825c-8206d54e5a28.png)
![Screenshot from 2023-04-17 17-53-31](https://user-images.githubusercontent.com/106893704/232608845-8fb7ecc0-643b-4a6e-9720-01b8270eeab7.png)

Reply:
@eerison about the footer, the social midia i left it like this just to be not empty, when add tabler into project i change

Reply:
@eerison when you get time for review, change the disk this forro is playing too much :p